### DISCUSSION TITLE
Clarification on `contains` query

### DISCUSSION THREAD

**Discussion by danielwinkler** [ravendb/ravendb#19414] [Q&A] [answered]
I have an index with entities that have a list of `locales`

If i perform a normal query then
`.Where(x => x.Locales.Contains("de-AT"))`
gives me a RQL
`where Locales = "de-AT"`

If i have a Document query, I have to write it as
`.ContainsAny(v => v.Locales, ["de-AT"])`
which translates to
`where Locales in ("de-AT")`

- Is there any difference between the query performance?
- Can i create the simpler contains query also for document queries?
- Is the contains query performing an extra search query using Lucene? in that case, would it be better to create a fanout index?

**Comment by ayende** [comment, marked as answer]
Both of them end up doing pretty much the same amount of work.
The Linq version is able to tell that you have just one time and simplified to `=`, the `DocumentQuery` can do that, but it doesn't actually matter that much.

You can use `WhereEquals(v=>v.Locals, "de-AT")` - yes.

You should _not_ use a fanout for this.

**Reply by danielwinkler** [reply]
Thanks for the quick response!
